fixture_magic/management/commands/custom_dump.py

```diff
@@ -38,10 +38,17 @@ def handle(self, *args, **options):
         dump_name = options['dump_name']
         pks = options['pk']
         dump_settings = settings.CUSTOM_DUMPS[dump_name]
-        (app_label, model_name) = dump_settings['primary'].split('.')
+        app_label, model_name, *subset = dump_settings['primary'].split('.')
         include_primary = dump_settings.get("include_primary", False)
-        dump_me = loading.get_model(app_label, model_name)
-        objs = dump_me.objects.filter(pk__in=[int(i) for i in pks])
+        manager = loading.get_model(app_label, model_name).objects
+        if subset:
+            dump_me = getattr(manager, subset[0])()
+        else:
+            dump_me = manager
+        if pks:
+            objs = dump_me.filter(pk__in=pks)
+        else:
+            objs = dump_me
```

Comment:
Let's use `queryset` instead of `dump_me` and `objs`. Makes it a bit more legible and saves an else.

Reply:
I was trying to keep the original naming sheme intact.